tox.ini

```diff
@@ -3,7 +3,7 @@ skip_missing_interpreters = True
 skipsdist = True
 
 envlist =
-    py{27,33,34,35}-dj{18,19,110}-wt{14,15,16}
+    py{27,33,34,35}-dj{18,19,110,111}-wt{14,15,16,17,18,19,20}
```

Comment:
I think we can ditch 1.4, 1.5, 1.6 and 1.7 since 1.8 is the Wagtail LTS.
In addition `20` isn't a thing 1.10 and 1.11 is. Please have a look at Wagtailtrans for this.